Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -1008,6 +1008,200 @@
+        <ErrorArray Error="CantBuildLocationInvalid" Sound="Alert_ZergBuildError" GroupSound="Alert_ZergBuildError"/>
+        <ErrorArray Error="CantLandLocationInvalid" Sound="Alert_ZergBuildError" GroupSound="Alert_ZergBuildError"/>
+        <GroupIcon>
+            <Image value="Assets\Textures\Wireframe-Zerg-NydusNetwork.dds"/>
+        </GroupIcon>
+        <HeroIcon value="Assets\Textures\btn-building-zerg-nydusnetwork.dds"/>
+        <SoundArray index="Ready" value="NydusNetwork_Ready"/>
+        <SoundArray index="What" value="NydusNetwork_What"/>
+        <UnitIcon value="Assets\Textures\btn-building-zerg-nydusnetwork.dds"/>
+        <Wireframe>
+            <Image value="Assets\Textures\Wireframe-Zerg-NydusNetwork.dds"/>
+        </Wireframe>
+    </CActorUnit>
+    <CActorModel id="AP_NydusNetworkBuild" parent="ZergBuildingBuild">
+        <Model value="NydusNetworkBuild"/>
+    </CActorModel>
+    <CActorSimple id="AP_NydusWormBuild">
+        <On Terms="ActorCreation" Send="Destroy"/>
+    </CActorSimple>
+    <CActorSimple id="AP_OmegaWormBuild">
+        <On Terms="ActorCreation" Send="Destroy"/>
+    </CActorSimple>
+    <CActorUnit id="NydusNetwork">
+        <On index="77" Terms="AbilTransport.AP_NydusWormTransport"/>
+        <On index="78" Terms="AbilTransport.AP_NydusWormTransport"/>
+    </CActorUnit>
+    <CActorModel id="NydusRumble">
+        <On Terms="UnitConstruction.AP_NydusWorm.Start" Send="Create"/>
+        <On Terms="UnitConstruction.AP_NydusWorm.Finish" Send="Destroy"/>
+        <On Terms="UnitConstruction.AP_NydusWorm.Cancel" Send="Destroy"/>
+    </CActorModel>
+    <CActorSound id="NydusWorm_Birth">
+        <On Terms="UnitBirth.AP_NydusWorm.Normal" Send="Create"/>
+        <On Terms="UnitBirth.AP_OmegaWorm.Normal" Send="Create"/>
+    </CActorSound>
+    <CActorSound id="NydusWorm_DeathCry">
+        <On Terms="UnitDeathCustomize.AP_NydusWorm; PassChance 100.000000" Send="Create"/>
+        <On Terms="UnitDeathCustomize.AP_OmegaWorm; PassChance 100.000000" Send="Create"/>
+        <On Terms="UnitDeathCustomize.AP_OmegaWorm; Death Salvage" Send="Destroy"/>
+        <On Terms="UnitDeathCustomize.AP_NydusWorm; Death Salvage" Send="Destroy"/>
+    </CActorSound>
+    <CActorSound id="Nydus_ConstructionFinish">
+        <On Terms="UnitConstruction.AP_NydusWorm.Finish" Send="Create"/>
+        <On Terms="UnitConstruction.AP_OmegaWorm.Finish" Send="Create"/>
+    </CActorSound>
+    <CActorSound id="Nydus_ConstructionStart">
+        <On Terms="UnitConstruction.AP_NydusWorm.Start" Send="Create"/>
+        <On Terms="UnitConstruction.AP_OmegaWorm.Start" Send="Create"/>
+    </CActorSound>
+    <CActorSound id="Nydus_Unburrow">
+        <On Terms="UnitBirth.AP_NydusWorm.Normal" Send="Create"/>
+        <On Terms="UnitBirth.AP_OmegaWorm.Normal" Send="Create"/>
+        <On Terms="Behavior.AP_SalvageNydus.On" Send="Create"/>
+        <On Terms="Behavior.AP_SalvageNydus.Off" Send="Destroy"/>

```

Comment:
Is it needed to patch vanilla stuff (if so, add `<!-- Override -->` comment, otherwise create a new Actor, dedicated to AP unit